observability/otel/otel-rn.adoc

```diff
@@ -12,9 +12,81 @@ You can use the {OTELName} xref:otel-forwarding-telemetry-data.adoc#otel-forward
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
-// git rebase marker 1
+[id="otel_3-4_{context}"]
+== Release notes for {OTELName} 3.4
 
-// git rebase marker 2
+The {OTELName} 3.4 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/opentelemetry-operator-bundle/615618406feffc5384e84400/history[{OTELOperator} 0.112.0].
+
+The {OTELName} 3.4 is based on the open source link:https://opentelemetry.io/docs/collector/[OpenTelemetry] release 0.112.0.
+
+////
+[id="otel_3-4_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-2024-????[CVE-2024-????]
+* link:https://access.redhat.com/security/cve/CVE-2024-????[CVE-2024-????]
+* link:https://access.redhat.com/security/cve/CVE-2024-????[CVE-2024-????]
+////
+
+[id="otel_3-4_technology-preview-features_{context}"]
+=== Technology Preview features
+
+This update introduces the following Technology Preview features:
+
+* OTLP JSON File Receiver
+* Count Connector
+
+:FeatureName: Each of these features
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="otel_3-4_new-features-and-enhancements_{context}"]
+=== New features and enhancements
+
+This update introduces the following enhancements:
+
+* The following link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features reach General Availability:
+** BearerTokenAuth Extension
+** Kubernetes Attributes Processor
+** Spanmetrics Connector
+* Multi-container auto-instrumentation injection for the OpenTelemetry SDK environment variables can be enabled by using the `instrumentation.opentelemetry.io/inject-sdk` annotation in the `Instrumentation` custom resource.
+
+////
+
+[id="otel_3-4_jaeger-release-notes_deprecated-functionality_{context}"]
+=== Deprecated functionality
+
+* In the {OTELName} 3.4, ???.
+* In the {OTELName} 3.4, ???.
```

Comment:
```suggestion
* In the {OTELName} 3.4, the Memory Ballast Extension was removed. Please use the Memory Limiter Processor instead.
```

cc @iblancasa can you confirm this is the proper replacement? From the README it looks like it.

There is also the memory limiter extension, but this ["is under development and does nothing"](https://github.com/open-telemetry/opentelemetry-collector/blob/eaeacaff2cfc7b9affd633bf1372fa8fdc8aaf03/extension/memorylimiterextension/README.md?plain=1#L17).

Reply:
@max-cx can you also remove the "Memory Ballast extension" from the docs here: https://docs.openshift.com/container-platform/4.17/observability/otel/otel-collector/otel-collector-extensions.html#memory-ballast-extension_otel-collector-extensions? it was in tech preview and got removed upstream now.

Reply:
It is setting the GOMEMLIMIT env var: some discussion here https://github.com/open-telemetry/opentelemetry-helm-charts/issues/891

Reply:
Thank you, both. I've updated the PR.